### DISCUSSION TITLE
FeatureRequest: Simple text field for annotations

### DISCUSSION THREAD

**Discussion by Jambola77** [arendst/Tasmota#11198] [Ideas]
I have a feature request for a simple text field for leaving some annotations etc.
I have several tasmota device where some of them have an individual image. To have a remainder when making an upgrade it would be helpful for leaving some notices inside the configuration or upgrade menu.

What do you think?

**Comment by ascillato** [comment]
For storing just a random text, you can use a MEM or a RULE turned off

**Reply by Jambola77** [reply]
Yes, but this is not displayed in the GUI.

**Reply by nagyrobi** [reply]
I think there'a a way to display virtual buttons

**Reply by sfromis** [reply]
When compiling, I think that you can use the #define CODE_IMAGE_STR setiing to supply an id for your own build, appearing in "Program Version" on the "Information" page.
